xmodule/modulestore/tests/django_utils.py

```diff
@@ -318,7 +318,7 @@ def start_modulestore_isolation(cls):
 
         cls.__old_modulestores.append(copy.deepcopy(settings.MODULESTORE))
         cls.__old_contentstores.append(copy.deepcopy(settings.CONTENTSTORE))
-        override.__enter__()
+        override.__enter__()  # pylint: disable=unnecessary-dunder-call
```

Comment:
Refactoring to use ContextManager `with` instead as `with open(override_settings(...)) as override` causes the unit tests to fail with the error `TypeError: expected str, bytes or os.PathLike object, not override_settings`. 
Detailed Error Stack Trace: https://github.com/openedx/edx-platform/actions/runs/3265468332/jobs/5367724316#step:7:3517